kafka-rest/src/main/java/io/confluent/kafkarest/CustomLogFields.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License
+ */
+
+package io.confluent.kafkarest;
+
+/**
+ * This class lists the fields that are added to the request-log(see RestCustomRequestLog.java).
+ * These field(& corresponding) value are stored as request-attribute.
+ */
+public class CustomLogFields {
```

Comment:
This should be name something like `CustomRequestAttributeFields` I think,

also, as this is a utility static class, it should be a final class, and there should be a private constructor to prevent instantiation.